src/components/Button/Button.tsx

```diff
@@ -19,44 +15,14 @@ export interface Props {
    * Disables the field and prevents editing the contents
    */
   disabled?: boolean;
-  /**
-   * Toggles button that fills the full width of its container
-   */
-  fullWidth?: boolean;
```

Comment:
See the comment on `block`.

Reply:
I'll revert this change and ask design whether we want to support this feature in this component: https://app.shortcut.com/czi-edu/story/188471/button-api-questions-for-design